validation-test/stdlib/POSIXErrorCodeLinux.swift

```diff
@@ -0,0 +1,53 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+//
+// UNSUPPORTED: OS=macosx
+// UNSUPPORTED: OS=ios
+// UNSUPPORTED: OS=tvos
+// UNSUPPORTED: OS=watchos
+// UNSUPPORTED: OS=linux-androideabi
+
+// REQUIRES: OS=linux-gnu
```

Comment:
This test can probably be generalized to more platforms with minimal changes.  Could you look into making it cross platform?